packages/server/src/utils/builders/heroku.ts

```diff
@@ -16,13 +16,14 @@ export const buildHeroku = async (
 		application.project.env,
 	);
 	try {
+		const builderVersion = env.HEROKU_STACK_VERSION || '24';
 		const args = [
 			"build",
 			appName,
 			"--path",
 			buildAppDirectory,
 			"--builder",
-			"heroku/builder:24",
+			`heroku/builder:${builderVersion}`,
```

Comment:
the env property is not a object, is a string in this format, this will not work I suggest to check the contributing.md https://github.com/Dokploy/dokploy/blob/canary/CONTRIBUTING.md#setup

```
ENV="something"
Something="something"
```